src/components/About/Clients.module.css

```diff
@@ -22,6 +22,7 @@
   left: 0;
   height: 100px;
   transform: translate(0, -45%);
+  z-index: 0;
```

Comment:
With mobile/table widths part of the image goes in the background and the directors section goes in the foreground.

Reply:
This is a reasonable change. However, there have been changes merged to main recently that are going to conflict with this because we changed from using img tags in the HTML for the decorative graphics to using `::before` and `::after` pseudo-elements with the images in the CSS. See the [new Clients.module.css implementation](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/blob/69ae621c3edc5a1a9430d2302f57661cc4224c4a/src/components/About/Clients.module.css#L19-L25).

You can revert this line, and ideally also update the Directors component to use the same pattern.